libopenage/renderer/camera/camera.h

```diff
@@ -94,7 +97,9 @@ class Camera {
 	 *              value is multiplied with the directional vector before its applied to
 	 *              the positional vector.
 	 */
-	void move_rel(Eigen::Vector3f direction, float delta = 1.0f);
+	void move_rel(Eigen::Vector3f direction, std::pair<float, float> x_bounds,
+											std::pair<float, float> z_bounds,
+											float delta = 1.0f);
```

Comment:
Same here as above:

- arguments should be documented in the doc string
- formatting should be done for the arguments
- you could assign default values here